lib/utils.py

```diff
@@ -31,25 +31,25 @@ def build_check(target: Path) -> bool:
     num_cpus = len(os.sched_getaffinity(0))
     shutil.copy('.config', '.tmp.config')
     try:
-        print("Building arm")
-        data = ["make", "ARCH=arm", "LLVM=1", "-j", str(num_cpus), "defconfig", target]
-        subprocess.check_output(data)
-        print("arm works")
-
-        print("Building arm64")
-        data[1] = "ARCH=arm64"
-        subprocess.check_output(data)
-        print("arm64 works")
-
-        print("Building riscv")
-        data[1] = "ARCH=riscv"
-        subprocess.check_output(data)
-        print("riscv works")
-
-        print("Building x86")
-        data[1] = "ARCH=x86"
-        subprocess.check_output(data)
-        print("x86 works")
+        clang_archs = ["arm", "arm64", "riscv", "powerpc", "x86"]
```

Comment:
we can add `ARCH=um` for clang (and s390 and mips). https://clangbuiltlinux.github.io/ has some more info.